reledmac.dtx

```diff
@@ -3314,23 +3314,23 @@
 %
 %
 % \subsection{Known and suspected limitations}
-% \subsubsection{Not standard geometry}
-% If you use class other than \ltxclass{article} or \ltxclass{book}, or if you use the geometry package, you should use \protect{maxhnotesX} and / or \cs{Xmaxhnotes} as explained in \reff{Xmaxhnotes} in order to prevent footnotes overlapping bottom margin.
+% \subsubsection{Non-standard geometry}
+% If you use classes other than \ltxclass{article} or \ltxclass{book}, or if you use the geometry package, you should use \protect{maxhnotesX} and\slash{}or \cs{Xmaxhnotes} as explained in \reff{Xmaxhnotes} in order to prevent footnotes from overlapping the bottom margin.
 %
 % \subsubsection{\package{floatrow} package compatibility}
 % The \package{floatrow} package must be loaded before the \macpackage.
 % \subsubsection{`No room for a new'}\label{noroom}
-% Sometime, especially when using \macpackage with other packages, you could obtain warning message such 'no room for a new count' or 'no room for a new write'.
+% Sometimes, especially when using \macpackage with other packages, you could obtain warning messages such 'no room for a new count' or 'no room for a new write'.
 %
-% The first thing in order to prevent such problem is to use the options to optimize \macpackage.
-% For example, if you need only two series of notes, use \verb+series={A,B}+ option.
-% Read \reff{optimize} in order to know which are there  options.
+% In order to prevent such problems, the first thing is to use the options to optimize \macpackage.
+% For example, if you need only two series of notes, use the \verb+series={A,B}+ option.
+% Read \reff{optimize} in order to know which are the available options.
 %
-% However, if with these options you still have such message, here are some tricks.
+% However, if with these options you still have such messages, here are some tricks.
 % \begin{description}
-%  \item['no room for a new count'] is often caused by a conjunction with \protect\package{biblatex}.
+%  \item['no room for a new count'] is often caused by a conjunction (** do you mean here `conflict'?) with \protect\package{biblatex}.
```

Comment:
no, it is not _stricto sensus_ a conflict, but just a little problem when using biblatex package in the same time